### ISSUE TITLE
TL is not always an extension of FOL

### ISSUE THREAD

**Issue by foysavas** [Dicklesworthstone/introduction_to_temporal_logic#1]
In fact, some of the most interesting Temporal Logics are not extensions of FOL.

It would be more proper to say that Temporal Logic is an extension of Propositional Logic with modal operator acting over paths. Those modal operators are analogous to the universal and existential operators found in FOL, but not the same. The most expressive Temporal Logics will have both pairs of dual operators.

Anyway, this is a great write-up. Happy to help you think through nuances.

**Comment by Dicklesworthstone** [comment]
Thanks for the clarification, I revised the text to reflect that it doesn't necessarily have to build on top of FOL.

**Comment by Dicklesworthstone** [comment]
Good point - temporal logic doesn't always extend first-order logic in a strict sense. The relationship depends on the specific formalism. This is an educational explainer, not a formal treatment, but the nuance is worth noting.
